lib/diagnostic.rb

```diff
@@ -105,15 +122,20 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < Cat
+  include Carnivorous
+  def roar
+    puts 'ROAR!'
+  end
+end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+# Mixins stores methods, is opening to all the classes.
```

Comment:
Classes can include any number of modules, but classes can inherit from only one other class.